security/pkg/server/ca/authenticate/kubeauth/kube_jwt.go

```diff
@@ -92,6 +96,20 @@ func (a *KubeJWTAuthenticator) Authenticate(ctx context.Context) (*security.Call
 	// tolerate the unbound tokens.
 	if !util.IsK8SUnbound(targetJWT) || security.Require3PToken.Get() {
 		aud = security.TokenAudiences
+		if tokenAud, _ := util.ExtractJwtAud(targetJWT); len(tokenAud) == 1 && tokenAud[0] == DefaultKubernetesAudience {
```

Comment:
nit: The bool returned isn't used. If it's not important it should be removed.

```suggestion
		if tokenAud, valid := util.ExtractJwtAud(targetJWT); valid && len(tokenAud) == 1 && tokenAud[0] == DefaultKubernetesAudience {
```

Reply:
its used in IsK8SUnbound but this is probably better